bu/CopcSupport.cpp

```diff
@@ -47,16 +46,20 @@ CopcSupport::CopcSupport(const BaseInfo& b) :
     m_header.creation.day = 1;
     m_header.creation.year = 1;
     m_header.header_size = lazperf::header14::Size;
-    m_header.vlr_count = 4;
-    m_header.point_format_id = b.opts.pointFormatId;
+    m_header.point_format_id = m_b.pointFormatId;
     m_header.point_format_id |= (1 << 7);    // Bit for laszip
-    m_header.point_record_length = lazperf::baseCount(b.opts.pointFormatId) + ebVLRSize();
+    m_header.point_record_length = lazperf::baseCount(m_b.pointFormatId) + extraByteSize();
     m_header.scale.x = b.scale[0];
     m_header.scale.y = b.scale[1];
     m_header.scale.z = b.scale[2];
     m_header.offset.x = b.offset[0];
     m_header.offset.y = b.offset[1];
     m_header.offset.z = b.offset[2];
+    m_header.vlr_count = 4;
```

Comment:
This means we are dropping other VLRs, right? That's fine, but maybe we should make a note in a doc or something.